docs/platforms/dart/integrations/http-integration.mdx

```diff
@@ -92,3 +92,19 @@ var uriResponse = await client.post('https://example.com/whatsit/create',
 
 await transaction.finish(status: SpanStatus.ok());
 ```
+
+## Using the `runWithClient` function
```

Comment:
Since this is a way to 'initialize' the integration, we can maybe push this up to the very top.

Maybe we can just add the code snippet as a tab like this: https://docs.sentry.io/platforms/kotlin-multiplatform/initialization-strategies/#usage

Reply:
moved it now to the top.